packages/app/tests/e2e/features/dashboard.spec.ts

```diff
@@ -4,6 +4,132 @@ test.describe('Dashboard', { tag: ['@dashboard'] }, () => {
   test.beforeEach(async ({ page }) => {
     await page.goto('/dashboards');
   });
+
+  test(
+    'should persist dashboard across page reloads',
+    { tag: '@full-stack' },
+    async ({ page }) => {
+      const uniqueDashboardName = `Test Dashboard ${Date.now()}`;
+
+      await test.step('Create and name a new dashboard', async () => {
```

Comment:
This test fails when running in local mode - (ie I ran `yarn test:e2e:ui`) - I see the tag for full stack, I wonder if theres  away to exclude the full stack tag in local mode (we may also need a way to add multiple tags, ex @dashboard and `@full-stack`)